src/XRP/default-xrp-client.ts

```diff
@@ -426,6 +428,27 @@ export default class DefaultXrpClient implements XrpClientDecorator {
     return XrpTransaction.from(getTransactionResponse, this.network)
   }
 
+  /**
+   * Enable Deposit Authorization for this XRPL account.
+   * @see https://xrpl.org/depositauth.html
+   *
+   * @param wallet The wallet associated with the XRPL account enabling Deposit Authorization and that will sign the request.
+   * @returns A promise which resolves to a string representing the hash of the submitted AccountSet transaction.
+   */
+  public async enableDepositAuth(wallet: Wallet): Promise<string> {
+    // 9 is the flag for enabling Deposit Auth: https://xrpl.org/accountset.html#accountset-flags
+    const setFlag = new SetFlag()
```

Comment:
Note: We have [`RippledFlags`](https://github.com/xpring-eng/Xpring-JS/blob/master/src/XRP/rippled-flags.ts) which are a set of transaction flags. 

We could maybe create a wrapper for this flag as well? Maybe `RippledFlags` should be renamed `TransactionFlags` to disambiguate? 

Open to this being a TODO with a follow on.

Reply:
Creating a separate, custom `AccountSetFlag` enum